docs/user_docs/installation/install-addons.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Install Addons
+description: Install KubeBlocks addons with Helm
+keywords: [addon, helm, KubeBlocks]
+sidebar_position: 4
+sidebar_label: Install Addons
+---
+
+# Install Addons
```

Comment:
and , we should hightligh somewhere  the major version of Addons and KB must be the same.